File: EmployeeAdminPortal/EmployeeAdminPortal/Models/Entities/Attendance.cs
namespace EmployeeAdminPortal.Models.Entities
{
    public class Attendance
    {
        public Guid Id { get; set; }
        public Guid EmployeeId { get; set; }
        public Employee? Employee { get; set; }
        public DateTime Date { get; set; }
        public DateTime? CheckIn { get; set; }
        public DateTime? CheckOut { get; set; }
    }
}

File: client/src/types/attendance.ts
export interface Attendance {
  id: string;
  employeeId: string;
  date: string;
  checkIn?: string | null;
  checkOut?: string | null;
}

export interface AddAttendanceDto {
  employeeId: string;
  date: string;
  checkIn?: string;
}

export interface UpdateAttendanceDto {
  checkOut: string;
}
